src/fosslight_util/download.py

```diff
@@ -127,14 +130,19 @@ def cli_download_and_extract(link, target_dir, log_dir, checkout_to="", compress
             link = src_info.get("url", "")
             tag = ''.join(src_info.get("tag",  "")).split('=')[-1]
             branch = ''.join(src_info.get("branch", "")).split('=')[-1]
+            is_rubygems = src_info.get("rubygems", False)
 
-            if not download_git_clone(link, target_dir, checkout_to, tag, branch):
+            # General download (git clone, wget)
+            if (is_rubygems is False) and (not download_git_clone(link, target_dir, checkout_to, tag, branch)):
```

Comment:
@bjk7119 , is False보다 not을 사용하는 것을 권장합니다.

You should know: False == 0 == None in case of if condition.
 If you use `if not`, you can cover all version of False (zero value). If you use `== False ` you cannot handle the 0 or None. **if not is recommended.** The is operator is a different story (is not same as ==) but you can read more details on this link: [Understanding Python's "is" operator](https://stackoverflow.com/questions/13650293/understanding-pythons-is-operator)

Reply:
Ref. https://peps.python.org/pep-0008/#programming-recommendations

![image](https://user-images.githubusercontent.com/4284712/223041523-7e9cf9fb-a9df-425d-a0d8-4db34bcc8ac0.png)